libshortfin/setup.py

```diff
@@ -71,16 +205,18 @@ def copy_extensions_to_source(self, *args, **kwargs):
     ],
     zip_safe=False,
     package_dir={
-        "_shortfin": str(python_src_dir / "_shortfin"),
-        "_shortfin_default": str(python_bin_dir / "_shortfin_default"),
+        "_shortfin": str(PYTHON_SOURCE_DIR / "_shortfin"),
+        "_shortfin_default": str(PYTHON_BINARY_DIR / "_shortfin_default"),
         # TODO: Conditionally map additional native library variants.
-        "shortfin": str(python_src_dir / "shortfin"),
+        "shortfin": str(PYTHON_SOURCE_DIR / "shortfin"),
     },
     ext_modules=[
-        BuiltExtension("_shortfin_default.lib"),
+        CMakeExtension("_shortfin_default.lib")
         # TODO: Conditionally map additional native library variants.
     ],
     cmdclass={
+        "build": CustomBuild,
```

Comment:
I think you may need to condition this somehow on is_cpp_prebuilt

Reply:
With is_cpp_prebuilt, the logic in CMakeBuildPy makes sure that we don't try to run cmake if we have prebuilt things.